github/mock/mock.go

```diff
@@ -0,0 +1,31 @@
+package mock
```

Comment:
Consider using the [mock that is recommended by go-github](https://github.com/google/go-github?tab=readme-ov-file#testing-code-that-uses-go-github).

Reply:
I created an issue for myself and I would explore this option later on and consider it, thanks!
https://github.com/jan-zabloudil/release-manager/issues/31